MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1189,8 +1189,7 @@ public string GetInputArgument(EncodingJobInfo state, EncodingOptions options, s
             {
                 var tmpConcatPath = Path.Join(_configurationManager.GetTranscodePath(), state.MediaSource.Id + ".concat");
                 _mediaEncoder.GenerateConcatConfig(state.MediaSource, tmpConcatPath);
-                arg.Append(" -f concat -safe 0 -i ")
-                    .Append(tmpConcatPath);
+                arg.Append(CultureInfo.InvariantCulture, $" -f concat -safe 0 -i \"{tmpConcatPath}\"");
```

Comment:
Either this or AppendFormat. We shouldn't be giving the stringbuilder interpolated strings.
```suggestion
                arg.Append(" -f concat -safe 0 -i \"")
                    .Append(tmpConcatPath")
                    .Append('"');
```